hdk-proc-macros/src/code_generators/zome_setup.rs

```diff
@@ -0,0 +1,23 @@
+use crate::zome_code_def::ZomeCodeDef;
+use proc_macro2::TokenStream;
+use quote::quote;
+
+impl ZomeCodeDef {
+    pub fn zome_setup(&self) -> TokenStream {
+        let entry_fn_idents = self
+            .entry_def_fns
+            .iter()
+            .map(|func| func.ident.clone())
+            .clone();
+
+        quote! {
+            #[no_mangle]
+            #[allow(unused_variables)]
+            pub extern "C" fn zome_setup(zd: &mut hdk::meta::ZomeDefinition) {
+                #(
+                    zd.define(#entry_fn_idents ());
```

Comment:
What's happening here where there's a space?
`#entry_fn_idents ()`
does that matter?

Reply:
Strangely if you expand the macro using `cargo expand` this is how it expands function calls be default. I think I did this the same way so the output is consistent.